src/outside_execution/outside_execution_hash.cairo

```diff
@@ -124,11 +124,15 @@ impl StructHashOutsideExecutionRev1 of IStructHashRev1<OutsideExecution> {
         while let Option::Some(call) = calls_span.pop_front() {
             hashed_calls.append(call.get_struct_hash_rev_1());
         };
+
+        let (nonce_channel, nonce_index) = self.nonce;
+
         poseidon_hash_span(
             array![
                 OUTSIDE_EXECUTION_TYPE_HASH_REV_1,
```

Comment:
We should update the TYPE_HASH to reglect new fields
```cairo
const OUTSIDE_EXECUTION_TYPE_HASH_REV_1: felt252 =
    selector!(
        "\"OutsideExecution\"(\"Caller\":\"ContractAddress\",\"Nonce\":\"felt\",\"Execute After\":\"u128\",\"Execute Before\":\"u128\",\"Calls\":\"Call*\")\"Call\"(\"To\":\"ContractAddress\",\"Selector\":\"selector\",\"Calldata\":\"felt*\")"
    );
```
->
```cairo
const OUTSIDE_EXECUTION_TYPE_HASH_REV_1: felt252 =
    selector!(
        "\"OutsideExecution\"(\"Caller\":\"ContractAddress\",\"Nonce\":\"(felt,felt)\",\"Execute After\":\"u128\",\"Execute Before\":\"u128\",\"Calls\":\"Call*\")\"Call\"(\"To\":\"ContractAddress\",\"Selector\":\"selector\",\"Calldata\":\"felt*\")"
    );
```
It's not critical but we should remain consistent